code/modules/racial_roundstartperks/racial_perk.dm

```diff
@@ -0,0 +1,253 @@
+///////////////////////////////////////Akula Based perks
+/datum/perk/recklessrage
+	name = "Reckless Rage"
+	desc = "Your body is powerful and strong when you succumb to instinct, but doing so leaves you without much higher reasoning for a short time. The rush of chemicals is also highly addictive \
+	and often times will leave your body weaker for a short time..... and such reckless rage makes you have a extreme tunnel vision and bright lights are extremely effective againts you. Such is everything else really. You are tough but weak."
+	active = FALSE
+	passivePerk = FALSE
+	var/init_sight
+	var/init_flash
+	var/obj/screen/lightOverlay = null
+
+/datum/perk/recklessrage/assign(mob/living/carbon/human/H)
+	..()
+	init_sight = holder.additional_darksight
+	init_flash = holder.flash_mod
+	holder.additional_darksight = -3
+	holder.flash_mod += 4
+
+/datum/perk/recklessrage/remove()
+	holder.additional_darksight = init_sight
+	holder.flash_mod = init_flash
+	..()
+/datum/perk/recklessrage/assign(mob/living/carbon/human/H)
```

Comment:
Why do you have two `assign()` procs separated? Wouldn't it be easier to just flatten all these modifiers into one?